src/sections/Projects/Sistent/components/link/index.js

```diff
@@ -0,0 +1,188 @@
+
+const SistentLink = () => {
+  const location = useLocation();
+  const { isDark } = useStyledDarkMode();
+
+  return (
+    <SistentLayout title="Link" >
+      <div className="content">
+        <a id="Identity">
+          <h2>link</h2>
+        </a>
+        <p>
+        A link component is a navigational element that directs users to another page or section within an application.
+
+        </p>
+        <div className="filterBtns">
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/link"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/link")}
+            title="Overview"
+          />
+          <TabButton
+            className={
+              location.pathname ===
+              "/projects/sistent/components/link/guidance"
+                ? "active"
+                : ""
+            }
+            onClick={() =>
+              navigate("/projects/sistent/components/link/guidance")
+            }
+            title="Guidance"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/link/code"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/link/code")}
+            title="Code"
+          />
+        </div>
+        <div className="main-content">
+          <p>
+            Links are essential and integral components of an interface. They are primarily used for navigation, guiding users to the next step in a journey or redirecting them to relevant sections or pages. Links can be paired with other elements to create cohesive designs, enhancing the digital experience by providing smooth transitions and interaction cues.
+          </p>
+          <a id="Types of Links">
+            <h2>Types of Links</h2>
+          </a>
+          <p>
+          Links in an interface serve as vital elements for navigation and interaction. Multiple types of links are available to establish hierarchy, provide visual prominence, and enhance flexibility, guiding users towards their goals efficiently. With distinct styling and behaviors, links can help streamline user flows and create a cohesive user experience.
+          </p>
+          <h3>Simple Links</h3>
+          <p>
+          Basic anchor links with customizable colors and typography styles that inherit properties from the Typography component. This allows flexibility in styling with options like various text sizes or variants.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Link
+                href="https://layer5.io/"
+                style={{
+                  marginLeft: "10px",
+                  color: "green",
```

Comment:
@NishantSinghhhhh don't pass any custom color, the link already has that required color which is the brand color.

Reply:
Sir I did that , but when no color was given , the link was rendering in black color and not green